app/components/containerConsole/containerConsoleController.js

```diff
@@ -39,14 +39,15 @@ function ($scope, $stateParams, Container, Image, Exec, $timeout, EndpointProvid
     $('#loadConsoleSpinner').show();
     var termWidth = Math.round($('#terminal-container').width() / 8.2);
     var termHeight = 30;
+    var command = $scope.state.isCustomCommand ? $scope.state.customCommand : $scope.state.command;
     var execConfig = {
       id: $stateParams.id,
       AttachStdin: true,
       AttachStdout: true,
       AttachStderr: true,
       Tty: true,
       User: $scope.state.user,
-      Cmd: $scope.state.command.replace(' ', ',').split(',')
+      Cmd: command.replace(' ', ',').split(',')
```

Comment:
The `ContainerHelper` already contains a method to handle custom string commands to array, I'd recommend using it to avoid strange behaviors (especially with `'` & `"` characters).

See `ContainerHelper.commandStringToArray(command_string);`